transmart-core-db/grails-app/services/org/transmartproject/db/support/SystemService.groovy

```diff
@@ -124,7 +134,7 @@ class SystemService implements SystemResource {
             'user query set scan': { ->
```

Comment:
`patientSetService.clearPatientSetIdsCache` above is done twice here - before and after refreshing the bitset materialized view (it is also a part of clearCaches()). Is it on purpose or I can remove one of them? @gijskant @forus

Reply:
One of them can go indeed.